CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.4.0 (2021-03-04)
+
+- Added the `getWidth`, `syncMaxHeight` and `resetHeight` methods to the `SkyCoreAdapterService`. [#213](https://github.com/blackbaud/skyux-core/pull/213)
```

Comment:
#213 Mentions adding methods **syncHeight**, resetHeight, and getWidth. Should syncHeight also be added to this description?

Reply:
Nice eye! The method was originally called `syncHeight` but we changed it in that PR to be `syncMaxHeight`.